docs/guides/guide-vmanomaly-vmalert.md

```diff
@@ -347,7 +359,10 @@ services:
     restart: always
     volumes:
       - ./vmanomaly_config.yml:/config.yaml
-    command: [ "/config.yaml" ]
+    platform: "linux/amd64"
+    command: 
+      - "/config.yaml"
+      - "--license=YOUR_LICENCE_CODE_IS_HERE"
```

Comment:
after this docker-compose `.yml`, we can add reference to our licensing options, i.e.
'see other ways of how to use license for vmanomaly [here](https://docs.victoriametrics.com/vmanomaly.html#licensing)'